src/main/java/uk/gov/hmcts/reform/civil/controllers/cases/DocumentController.java

```diff
@@ -27,13 +33,27 @@
 public class DocumentController {
 
     private final ClaimFormService claimFormService;
+    private final DocumentManagementService documentManagementService;
 
     @PostMapping("/generateSealedDoc")
     public CaseDocument uploadSealedDocument(
         @RequestHeader(HttpHeaders.AUTHORIZATION) String authorisation, @NotNull @RequestBody CaseData caseData) {
         return claimFormService.uploadSealedDocument(authorisation, caseData);
     }
 
+    @PostMapping(value = "/generateAnyDoc")
+    @Operation(summary = "Upload document")
+    public CaseDocument uploadAnyDocument(
+        @RequestHeader(HttpHeaders.AUTHORIZATION) String authorisation,
+        @RequestParam("file")MultipartFile file) {
+        try {
```

Comment:
my suggestion in here is:
public ResponseEntity<CaseDocument>uploadAnyDocument(
        @RequestHeader(HttpHeaders.AUTHORIZATION) String authorisation,
        @RequestParam("file")MultipartFile file) {
UploadedDocument uploadedDocument = new UploadedDocument(file.getOriginalFilename(), file.getBytes());
            return ResponseEntity.ok(documentManagementService.uploadDocument(authorisation, uploadedDocument);
            }